packages/plugins/rubygem-foreman_ovirt/rubygem-foreman_ovirt.spec

```diff
@@ -4,13 +4,17 @@
 %global foreman_min_version 3.16
 
 Name: rubygem-%{gem_name}
-Version: 2.0.0
+Version: 2.0.1
 Release: 1%{?foremandist}%{?dist}
 Summary: oVirt as a compute resource for The Foreman
 License: GPLv3
 URL: https://github.com/markt-de/foreman_ovirt
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
+# Foreman 3.16 dropped the oVirt compute provider and this is the replacement
+Obsoletes: foreman-ovirt < 3.16.0
```

Comment:
This is also present in `foreman`:
https://github.com/theforeman/foreman-packaging/blob/94f7f3017bef91098483aefd757fd3925847a1e4/packages/foreman/foreman/foreman.spec#L28-L29

Wonder what RPM/DNF will do with this?

Reply:
But I could test it, when new RPMs are available.

Reply:
It's probably best to drop it on Foreman itself. Then on the downstream side we should solve it in a different way.